mrtt-ui/package.json

```
@@ -18,7 +18,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "jest",
```

Using `react-scripts test` here works so let's stick with that.
```suggestion
    "test": "react-scripts test",
```

---

ah yes, works both ways. Just reverted back to` react-scripts test`